### PR TITLE
[MNG-6859] Build not reproducible when built from source release

### PR DESCRIPTION
**Pull request by michael-o**
This closes #322

### CODE REVIEW

File: maven-core/pom.xml [hunk 1]
```diff
@@ -164,31 +164,6 @@ under the License.
             </excludes>
           </configuration>
         </plugin>
-        <!--This plugin's configuration is used to store Eclipse m2e settings only. It has no influence on the Maven build itself.-->
```
Comment (eolivelli) [inline maven-core/pom.xml]: Is this change related to the issue somehow?

Comment (slachiewicz) [reply maven-core/pom.xml]: It's obsolete as we no longer use create-timestamp goal.

Comment (michael-o) [reply maven-core/pom.xml]: @slachiewicz Correct.

**Comment by jira-importer** [comment]
Resolve #8440
